servicecontrol/upgrades/3to4/index.md

```diff
@@ -89,15 +91,16 @@ If a secondary instance has error ingestion turned off, then it cannot be upgrad
   - Audit log queue
   - Audit forwarding
   - Audit retention period
-- Turn off audit ingestion on the secondary instance by setting `ServiceBus/AuditQueue` to `!disable`. 
-- Create a new ServiceControl Audit instance with configuration that matches the settings in the original instance
-  - Use the `New-ServiceControlAuditInstance` powershell cmdlet
-- Add the new ServiceControl Audit instance to the `ServiceControl/RemoteInstances` setting in the primary ServiceControl instance
-  - Use the `Get-AuditInstances` powershell cmdlet to find the Url of the new secondary instance
-  - Use the `Add-ServiceControlRemote` powershell cmdlet 
-- Wait for the audit retention period to pass. This will allow all of the audited messages in the old secondary instance to get cleaned up
+  - Database folder
 - Remove the original secondary instance from the `ServiceControl/RemoteInstances` setting in the primary ServiceControl instance
```

Comment:
This is actually not needed when the port and db port would be reused as well right?

Reply:
Yeah. I couldn't find a succinct way of doing that. Maybe I should add Host and Port to the above list of config items to copy over and then we can assume it.

Reply:
I did that. It looks simpler :)